tests/blackbox/src/evaluation/scenario/scenario.py

```diff
@@ -187,7 +187,7 @@ def load_all_namespace_scope_scenarios(self, path: str, logger: Logger) -> None:
 
         # loop over all the directory names
         for dir_name in directories:
-            scenario_file = path + "/" + dir_name + "/scenario.yml"
+            scenario_file = path + "/" + dir_name + "/evaluation.yml"
```

Comment:
(nit): I would prefer to keep this file name as `scenario.yml`, as it more resembles as a test case.

Reply:
That works for me as well. Will undo the changes.